test/plugin/scenarios/netty-socketio-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {netty-socketio-scenario: 2}
+  instances:
+    - {netty-socketio-scenario: 1}
+  operationNames:
+    - netty-socketio-scenario: [/netty-socketio-scenario/case/netty-socketio, /socket.io/,
+                                /netty-socketio-scenario/healthCheck, SocketIO/onConnect, SocketIO/get_data/send,
+                                SocketIO/send_data/receive]
```

Comment:
What is `/socket.io/`? Others make sense to me.

Reply:
It's socket.io protocol request, need use HTTP request to upgrade to web socket.
Here I found some link to introduce this protocol, you can have a look.
https://stackoverflow.com/questions/38309640/how-socket-io-works
https://www.jianshu.com/p/03f1f45021cc

maybe I need to delete this /socket.io/ span?

Reply:
If it makes sense to user, I am good. Just for recheck.